### ISSUE TITLE
moe_lb_loss should be divided by config.gradient_accumulation_steps for reporting.

### ISSUE THREAD

**Issue by bzantium** [AI-Hypercomputer/maxtext#1483]
https://github.com/AI-Hypercomputer/maxtext/blob/db89bbb818e7dc98ee9d3ad14db0a24f436f0c55/MaxText/train.py#L468

moe_lb_loss should be divided by config.gradient_accumulation_steps for reporting.
```py
  moe_lb_loss = aux["moe_lb_loss"] / config.gradient_accumulation_steps
```

**Comment by RissyRan** [comment]
Thanks for reaching out! I think the gradient_accumlation_steps is handled [here](https://github.com/AI-Hypercomputer/maxtext/blob/db89bbb818e7dc98ee9d3ad14db0a24f436f0c55/MaxText/train.py#L451-L454)? Inside of `if config.gradient_accumulation_steps > 1`.


https://github.com/AI-Hypercomputer/maxtext/blob/db89bbb818e7dc98ee9d3ad14db0a24f436f0c55/MaxText/train.py#L451-L454

**Comment by bzantium** [comment]
you are right for loss calculation but not for moe lb loss logging on tensorboard.
https://github.com/AI-Hypercomputer/maxtext/blob/db89bbb818e7dc98ee9d3ad14db0a24f436f0c55/MaxText/train.py#L468-L486

in L485, current code provides `moe_lb_loss` which is summed through gradient accumulation if `config.gradient_accumulation_steps>1` following:
 https://github.com/AI-Hypercomputer/maxtext/blob/db89bbb818e7dc98ee9d3ad14db0a24f436f0c55/MaxText/train.py#L456

so need to fixed as suggested above.

**Comment by sarunsingla11722** [comment]
We are currently closing stale issues as part of a cleanup initiative. If any of these are still necessary, please feel free to reopen them.
